Separates Rock The Vote Import into two jobs to enable larger file imports

chloealee opened the pull request:
#### What's this PR do?
Separates Rock The Vote Import into two jobs to enable larger file imports.
 - One job to process CSV.
 - One job to create post in Rogue.

#### How should this be reviewed?
- Does all the logic seem ok? 

#### Any background context you want to provide?
- Unfortunately, we will have to get rid of the stats when we separate these jobs. Are we ok with this for now? We should still have all the info. in the logs though! 

#### Relevant tickets

Fixes [this](https://www.pivotaltracker.com/n/projects/2019429/stories/159660724) Pivotal Card.

katiecrane commented on app/Jobs/CreateTurboVotePostInRogue.php:
```diff
@@ -55,7 +54,7 @@ public function handle(Rogue $rogue)
             }
 
-            if ($user) {
+            if (isset($user)) {
```
good catch! thanks for explaining :)

katiecrane commented on app/Jobs/ImportRockTheVotePosts.php:
```diff
@@ -26,12 +26,10 @@ class ImportRockTheVotePosts implements ShouldQueue
 
     /**
-     * The path to the stored csv.
+     * The total records in the stored csv.
```
is this a count of the total records or is it the records themselves?

chloealee replied:
@katiecrane great q! This is [the count](https://github.com/DoSomething/chompy/blob/7b32036b779ce702f50076bc1e60c8c87c75554f/app/Traits/ImportToRogue.php#L35) of the total records. Updated this comment to make it more clear!

katiecrane commented on app/Jobs/CreateRockTheVotePostInRogue.php:
```diff
@@ -0,0 +1,392 @@
+                        $post = $rogue->createPost($postData);
+
+                        if ($post['data']) {
+                            info('post created in rogue for ' . $this->record['id']);
```
there's no "id" column in RTV csvs, I believe previously we were using email address?

chloealee replied:
ah great catch! Updated :)

katiecrane commented:
@chloealee it's really hard to tell if any of the logic changed in the fresh RTV file, I'm trying to figure out a good way to compare it to the old logic - do you know if you changed anything after you copied and pasted?

chloealee commented:
@katiecrane ah yes it is hard to tell ! You could maybe compare to [this file](https://github.com/DoSomething/chompy/blob/master/app/Jobs/ImportRockTheVotePosts.php)?

chloealee commented:
@katiecrane the only thing that I added was that log line that you caught before that had `$record['id']` instead of `$record['Email address']` and deleted the stats stuff since it won't work to have in separate two jobs!

katiecrane commented:
@chloealee okay I think we're good then! Let's just make sure to do some spot checking on QA just in case! great work!

chloealee commented:
yes definitely @katiecrane ! :)
